test/integration_tests/test_huggingface_udfs_ner.py

```diff
@@ -0,0 +1,62 @@
+# coding=utf-8
```

Comment:
This test case can be mvoed to the huggingface test file. Do not need a separate file for ner.

Reply:
made this changes and pushed and the test case name is test_named_entity_recognition_model_all_pdf_data